server/static/css/notebook.css

```diff
@@ -15,10 +15,6 @@
     width: inherit !important;
 }
 
-.nb-markdown-cell * {
-    font-family: Helvetica Neue, Helvetica, sans-serif !important;
```

Comment:
This messed with KaTeX rendering of text. I just opted to remove it and allow notebook fonts to match the surrounding page.